### ISSUE TITLE
Panel: Show Desktop: Switching button to Alternative "Minimze All" does minimize any open windows on first click, but not bring them back on second click?!

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.9#224]
tested on valve iso 20th febr.

**Comment by zzag** [comment]
Hmm, it seems like the Show Desktop applet does nothing to unminimize windows.

https://cgit.kde.org/plasma-desktop.git/tree/applets/showdesktop/plugin/showdesktop.cpp#n40

**Comment by star-buck** [comment]
But didnt it do before? I clearly remember us dicussing the usecase and me
trying it out to confirm working (as it does in Netrunner plasma 5.14.5)...
So it is either a bug or missing feature.

**Comment by Pointedstick** [comment]
I can't reproduce the issue on my personal machine (it un-minimizes windows just fine, see https://bugs.kde.org/show_bug.cgi?id=413044), but I can reproduce the issue on the Valve ISO. I wonder if our Valve-specific KWin stuff broke it.

**Comment by star-buck** [comment]
Whats Valve specific atm?

**Comment by davidedmundson** [comment]
There is nothing Valve specific that would affect the minimise all applet.

**Comment by davidedmundson** [comment]
>Hmm, it seems like the Show Desktop applet does nothing to unminimize windows.

Clemens is saying he's switched from Show Desktop to Minimize All.

**Comment by Pointedstick** [comment]
Yes, I did too.

**Comment by star-buck** [comment]
Yes, via right-click on the applet icon and choosing "Alternatives".

**Comment by davidedmundson** [comment]
> There is nothing Valve specific that would affect the minimise all applet.

I was wrong.

Was the workaround to make previews for minimised windows in the (old) dash.

Fixed.

**Comment by zzag** [comment]
> > Hmm, it seems like the Show Desktop applet does nothing to unminimize windows.
> 
> Clemens is saying he's switched from Show Desktop to Minimize All.

Sorry, I got confused.
